tools/bqtag/README.md

```diff
@@ -0,0 +1,374 @@
+```
+
+`bq_dataset` (required) is the BigQuery Dataset where Table and Authorized Views have to be created. 
+
+`catalog_taxonomy` (required) is the Data Catalog taxonomy name which holds the Policy Tags. Taxonomy may not be present and can be created using the create_taxonomy() function.
+
+`location` (required) is the location where BigQuery and DataCatalog resources are present or would be created.
+
+`bq_project` (optional - can be derived from authenticated service account) is the project where BigQuery resources are present or would be created.
+
+`catalog_project` (optional - can be derived from authenticated service account) is the project where Data Catalog resources are present or would be created.
+
+`json_credentials_path` (optional) path to service account credentials file. 
+
+
+
+## Create Taxonomy and Policy Tags
+
+Data Catalog Taxonomy and Policy Tags can be created if not already present. If Taxonomy and Policy Tags are already present then just specify the display name of Taxonomy in `catalog_taxonomy` variable and relevant policy tags would be downloaded from the Taxonomy. If, however, Taxonomy and Policy Tags are not present then `create_taxonomy()` function can be used to provision the same. Taxonomy name provisioned would be taken from the `catalog_taxonomy` variable. `create_taxonomy()` takes the following parameters:
+
+`tags` list of dictionaries where each dictionary describes a Policy Tag to be created. Each tag can have following attributes:
+
+- `name` (required) is the name of the Policy Tag to create
+- `description` (optional) is the description of the Policy Tags
+- `parent_policy_tag` (optional) is the link to the parent policy tag
+
+```python
+bqtv.create_taxonomy([
+                      {"name": "low", "description": "Low tag"}, 
+                      {"name": "medium", "description": "Medium tag"}, 
+                      {"name": "high", "description": "High tag"}
+                    ])
+```
+
+The function returns `True` if Taxonomy and Policy Tags are created successfully. If any error or exception is encountered, function will return `False`.
+## Fetch Policy Tags
+
+Before any Table or View can be created, Policy Tags have to be fetched from Data Catalog so that the complete Policy Tag name can be mapped to friendly names of policy tags. This can be done by calling the `fetch_policy_tags()` function. This function does not take any parameters.
+
+The function returns `True` if Policy Tags are fetched successfully. If any error or exception is encountered, function will return `False`.
+
+```python
+status = bqtv.fetch_policy_tags()
+if status:
+    # Rest of the code
+    # Rest of the code
+```
+
+## Create a Tagged Table
+
+`create_table()` function can create tagged tables inside a dataset in BigQuery. Dataset and BigQuery Project, defined at the time of initialization of the object is used as the dataset and project for the new table. This function takes the following attributes:
```

Comment:
The `create_table()` function...

Reply:
changed to:

The `create_table()` function can create tagged tables